cmd/ec2.go

```diff
@@ -15,21 +19,52 @@ var ec2Cmd = &cobra.Command{
 	Short: "Print ec2 related information",
 	Long: `For example,
 		awsctl get ec2`,
-	Run: func(cmd *cobra.Command, args []string) {
-		fmt.Println("ec2 called")
-	},
+	Run: getEC2command,
 }
 
-func init() {
-	getCmd.AddCommand(ec2Cmd)
+type EC2DesAPI interface {
+	DescribeInstances(ctx context.Context,
+		params *ec2.DescribeInstancesInput,
+		optFns ...func(*ec2.Options)) (*ec2.DescribeInstancesOutput, error)
+}
+
+// Returns info about ec2 instance
+func GetEc2Info(ctx context.Context, input *ec2.DescribeInstancesInput, api EC2DesAPI) (*ec2.DescribeInstancesOutput, error) {
```

Comment:
can we move function to `pkg` if possible. I will cover this in upcoming stream as well.

Reply:
Done